Backendless/Transaction/RelationOperation.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BackendlessAPI.Transaction
+{
+  interface RelationOperation
```

Comment:
do we need an interface? How many implementations of this interface are there? If there is only one, the interface is not really needed